src/opaque_closure.c

```diff
@@ -8,30 +8,38 @@ jl_value_t *jl_fptr_const_opaque_closure(jl_opaque_closure_t *oc, jl_value_t **a
     return oc->captures;
 }
 
-// TODO: remove
-jl_value_t *jl_fptr_va_opaque_closure(jl_opaque_closure_t *oc, jl_value_t **args, size_t nargs)
+// determine whether `argt` is a valid argument type tuple for the given opaque closure method
+JL_DLLEXPORT int jl_is_valid_oc_argtype(jl_tupletype_t *argt, jl_method_t *source)
```

Comment:
This seems to assume that `argt` is a datatype, without checking?

Reply:
`jl_new_opaque_closure` checks `jl_is_tuple_type`, and the other caller in codegen does as well.